src/main/java/me/zeroeightsix/fiber/tree/Node.java

```diff
@@ -54,27 +54,14 @@ default TreeItem lookup(String name) {
      * @param item The child to add
      * @return the child
      * @throws FiberException if there was already a non-transparent child by the same name or if {@code item} was a non-property item with the same name as a transparent item.
-     * @see Transparent
      * @see Property
      */
     default TreeItem add(@Nonnull TreeItem item) throws FiberException {
         TreeItem existing = lookup(item.getName());
         if (existing == null) {
             getItems().add(item);
         } else {
-            if (existing instanceof Transparent) {
-                if (item instanceof Property<?>) {
-                    Class<?> type = ((Property<?>) item).getType();
-                    // cannot add private helper methods to interfaces
-                    ((Property) item).setValue(((Transparent) existing).marshall(type));
-                    getItems().remove(existing);
-                    getItems().add(item);
-                } else {
-                    throw new FiberException("Attempt to replace transparent node by non-property node " + item.getName());
-                }
-            } else {
-                throw new FiberException("Attempt to replace non-transparent node " + existing.getName());
-            }
+            throw new FiberException("Attempt to replace non-transparent node " + existing.getName());
```

Comment:
Because transparent items are no longer a thing, consider changing this to `Attempt to replace existing item`?

Reply:
Correct, I did this a bit too quick.